### ISSUE TITLE
adding style in seed.less

### ISSUE THREAD

**Issue by mahirpiyarali** [mahirpiyarali/LaunchPad-D1#6]
I'm trying to create a better caption style (font size, italic, san-serif) in seed.less and trying to then apply it into my html. Can you take a look and see why it isn't showing? @thomaswilburn

**Comment by thomaswilburn** [comment]
I don't think you have your changes checked in. I would make sure that your selector is correct, and that it's not accidentally nested in something else.

**Comment by mahirpiyarali** [comment]
i added it into the html, just a small aesthetic question: is the letting between the lines of the caption too big? @thomaswilburn

**Comment by thomaswilburn** [comment]
In general, you shouldn't style using elements like `<small>`, you should apply CSS styles to the `.caption` selector in your seed.less file. You should also move the caption text into the caption div--right now it follows the div, which you can see because the closing tag (`</div>`)comes before it.

Ideally, your HTML should look something like this:

```html
<div class="caption">
caption text here
</div>
```

And your CSS should look like this:

```css
.caption {
  font-size: 15px;
  text-style: italic;
}
```

The very large line spacing is happening because you have the caption currently in an `<i>` element, which is an inline text type, and so it's treating the lines as very large for some reason. If you move it inside of the div, which is a block type, it'll wrap properly.
